src/benchmarks/benchdiff

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/perl -w
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This probably won't fly. A full license text is unfortunately required.

Reply:
This sucks.  I hoped to sneak it through — it feels wrong to have a whole page of legalese for a 2 pages code + 1 page table script, when the same information is conveyed by a single line in a standardized way.  But yeah, not a battle to wage right now.